nomos-blend/scheduling/src/message_blend/provers/leader/tests.rs

```diff
@@ -51,7 +51,7 @@ async fn proof_generation() {
     // Next proof should still return `Some` since leadership proofs do not have a
     // maximum cap.
     timeout(
-        Duration::from_secs(3),
+        Duration::from_secs(5),
```

Comment:
Slow CI, particularly on Linux runners, make this test fail sometimes, so bumping it by a bit more should be fine,